bundles/org.openhab.binding.nikohomecontrol/src/main/java/org/openhab/binding/nikohomecontrol/internal/handler/NikoHomeControlActionHandler.java

```diff
@@ -217,8 +217,6 @@ public void initialize() {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED,
```

Comment:
Additional improvement opportunity here: **BRIDGE_UNINITIALIZED** should be set by framework in combination with status **UNINITIALIZED**. When bridge is not initialized, `initialize()` in child things will not be called. Instead you can set **BRIDGE_OFFLINE** (if this is the case). For reference: https://www.openhab.org/docs/concepts/things.html#status-details

Reply:
@jlaur Correct, and this should not happen anymore at all. nhcComm cannot be null if the bridge has been initialized. I modified the ThingStatusDetail anyway.

Reply:
It will also return null in case `getBridgeHandler()` returns null. This can occur in case of misconfiguration where thing is configured without a bridge. Please check if this is now handled correctly - **CONFIGURATION_ERROR** would be more correct in this scenario.

Reply:
See for example:
https://github.com/openhab/openhab-addons/blob/88e7ba3774045325c6e85c613eeefb468db2ecef/bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewShadeHandler.java#L97-L102

Reply:
@jlaur Indeed, I didn't think that through properly for configuration errors. I changed it.